### PR TITLE
Add Tailwindcsss ,Vue and Svelte structured properly with `Learn with Sumit` resources

### PR DESCRIPTION
**Pull request by Mdkawsarislam2002**
### Add 10 Bangla Courses 

 * Vue.JS
 * Tailwindcsss 
 * Svelte 

### Why is this valuable (or not)?
 In Bengal resources, that's not enough, so I added the 2most helpful free resources And 

### How do we know it's really free?
 These resources are from YouTube for its totally free 

## Checklist:
* [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
* [x] Search for duplicates.
* [x] Include author(s) and platform where appropriate.
* [x] Put lists in alphabetical order, correct spacing.
* [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

* Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-bn.md [hunk 1]
```diff
@@ -135,6 +142,17 @@
 * [React JS Tutorial Bangla Series for Beginners](https://www.youtube.com/playlist?list=PLHiZ4m8vCp9M6HVQv7a36cp8LKzyHIePr) - Sumit Saha (Learn with Sumit)
 
 
+#### Vue
+
+* [Introduction to Vue.js in Bangla বাংলা  with a full Project - Vue Crash Course](https://youtu.be/iIvN7upsLoA) - Learn with Sumit - LWS - Bangladesh
```
Comment (LuigiImVector) [inline courses/free-courses-bn.md:147]:
```suggestion
* [Introduction to Vue.js in Bangla বাংলা  with a full Project - Vue Crash Course](https://www.youtube.com/watch?v=iIvN7upsLoA) - Sumit Saha
```
Shortened URLs are not allowed

File: courses/free-courses-bn.md [hunk 2]
```diff
@@ -90,6 +92,11 @@
 * [Bootstrap 5 and 4 Bangla Tutorials](https://www.youtube.com/playlist?list=PLgH5QX0i9K3oC_wmWEZa2xWxJauIRQ9kG) - Anisul Islam
 
 
+#### Tailwind
+
+* [Tailwind CSS Bangla Tutorial Series](https://youtube.com/playlist?list=PLHiZ4m8vCp9P23SqlHL0QAqiwS_oCofV2) - Learn with Sumit - LWS - Bangladesh
```
Comment (LuigiImVector) [inline courses/free-courses-bn.md:97]:
```suggestion
* [Tailwind CSS Bangla Tutorial Series](https://youtube.com/playlist?list=PLHiZ4m8vCp9P23SqlHL0QAqiwS_oCofV2) - Sumit Saha
```

**Review by LuigiImVector — CHANGES_REQUESTED** [review]

**Comment by eshellman** [comment]
This is a judgment call. We could use "Learn With Sumit" or "Sumit Saha". Depends on how the creator uses the name. If for example, emails from the creator are signed "Learn with Sumit", then that's an indication of how the creator wants to be identified, especially if "Learn With Sumit" is more than just Sumit. If all the videos start with "Hello I'm Sumit Suha", then clearly the creator is "Sumit Suha". If a user needs to work to discover the "Suha" part, then "Learn with Sumit" more clearly identifies the creator for potential students. I doubt the verbose "Learn with Sumit - LWS - Bangladesh" is an appropriate thing to list as the creator. So @Mdkawsarislam2002 please consider the suggestions and use the phrase that you think best identifies the creator, and thank you for bearing with us!

**Comment by Mdkawsarislam2002** [comment]
**Thanks.** 
Now it's ok, I think?

**Comment by LuigiImVector** [comment]
Checking the CONTRIBUTING file
https://github.com/EbookFoundation/free-programming-books/blob/a7558f56b8ffcd27571a8ff3fa4d6d11538a0755/docs/CONTRIBUTING.md#L163
and course listings there are no short youtube links (youtu.be)
I think it could be discussed whether to accept official short links or not.

File: courses/free-courses-bn.md [hunk 3]
```diff
@@ -135,6 +142,17 @@
 * [React JS Tutorial Bangla Series for Beginners](https://www.youtube.com/playlist?list=PLHiZ4m8vCp9M6HVQv7a36cp8LKzyHIePr) - Sumit Saha (Learn with Sumit)
 
 
+#### Vue
+
+* [Introduction to Vue.js in Bangla বাংলা  with a full Project - Vue Crash Course](https://youtu.be/iIvN7upsLoA) - Learn with Sumit
```
Comment (eshellman) [inline courses/free-courses-bn.md:147]: We do NOT accept shortened links.

File: courses/free-courses-bn.md [hunk 4]
```diff
@@ -135,6 +142,17 @@
...
+
+
+#### Svelte
+
+* [Svelte 3: The Game Changer (Bangla Crash Course)](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDJOOYnZxK-fbKxjxFHfc-H) - Stack Learner
+* [Svelte Bangla Overview Tutorial with examples - How Svelte is different from React & others](https://youtu.be/DdFcbIsqPh8) - Learn with Sumit
```
Comment (eshellman) [inline courses/free-courses-bn.md:153]: We do NOT accept shortened links.

**Review by eshellman — CHANGES_REQUESTED** [review]

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-bn.md [hunk 5]
```diff
@@ -135,6 +142,17 @@
...
 
+#### Vue
+
+* [Introduction to Vue.js in Bangla বাংলা  with a full Project - Vue Crash Course](https://www.youtube.com/watch?v=iIvN7upsLoA&list=PLHiZ4m8vCp9O5_QoMvQbOAOR5WolyZ4OS&index=7&t=1989s) - Learn with Sumit
+
+
+#### Svelte
+
+* [Svelte 3: The Game Changer (Bangla Crash Course)](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDJOOYnZxK-fbKxjxFHfc-H) - Stack Learner
+* [Svelte Bangla Overview Tutorial with examples - How Svelte is different from React & others](https://www.youtube.com/watch?v=DdFcbIsqPh8&list=PLHiZ4m8vCp9MrJGkaQGLJHapIz5goTqgl&index=11&t=820s) - Learn with Sumit
```
[the remark this answers is not in the document]
Comment (Mdkawsarislam2002) [reply courses/free-courses-bn.md:153]: ???

File: courses/free-courses-bn.md [hunk 6]
```diff
@@ -135,6 +142,16 @@
...
 
 
+#### Vue
+
+* [Introduction to Vue.js in Bangla বাংলা  with a full Project - Vue Crash Course](https://www.youtube.com/watch?v=iIvN7upsLoA&list=PLHiZ4m8vCp9O5_QoMvQbOAOR5WolyZ4OS&index=7&t=1989s) - Learn with Sumit
+
+
+#### Svelte
+
+* [Svelte 3: The Game Changer (Bangla Crash Course)](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDJOOYnZxK-fbKxjxFHfc-H) - Stack Learner
```
Comment (LuigiImVector) [inline courses/free-courses-bn.md:152]:
```suggestion
* [Explore Programming in Bangla ( বাংলা )](https://www.youtube.com/playlist?list=PLHiZ4m8vCp9MrJGkaQGLJHapIz5goTqgl) - Learn with Sumit
* [Svelte 3: The Game Changer (Bangla Crash Course)](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDJOOYnZxK-fbKxjxFHfc-H) - Stack Learner
```

**Review by LuigiImVector — CHANGES_REQUESTED** [review]
in the url after the question mark ? there are parameters separated by the symbol &, you have to keep only the essential parameters, in this case the ones that redirect to the video/playlist

this required @eshellman

File: courses/free-courses-bn.md [hunk 7]
```diff
@@ -10,18 +10,20 @@
...
 * [HTML and CSS](#html-and-css)
   * [Bootstrap](#bootstrap)
+  * [Tailwind](#Tailwind)
 * [IDE and Editors](#ide-and-editors)
 * [Java](#java)
 * [JavaScript](#javascript)
   * [jQuery](#jquery)
   * [Node.js](#nodejs)
   * [React](#react)
+  * [Vue](#Vue)
```
Comment (eshellman) [inline courses/free-courses-bn.md:20]: use lower case in fragment ids

File: courses/free-courses-bn.md [hunk 8]
```diff
@@ -10,18 +10,20 @@
 * [Go](#go)
 * [HTML and CSS](#html-and-css)
   * [Bootstrap](#bootstrap)
+  * [Tailwind](#Tailwind)
```
Comment (eshellman) [inline courses/free-courses-bn.md:13]: use lower case in fragment ids

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-bn.md (hunk 7 above)
Comment (davorpa) [reply courses/free-courses-bn.md:20]:
```suggestion
  * [Vue](#vue)
```

File: courses/free-courses-bn.md (hunk 8 above)
Comment (davorpa) [reply courses/free-courses-bn.md:13]: Patch suggested by Eric

```suggestion
  * [Tailwind](#tailwind)
```

**Comment by davorpa** [comment]
Conflicts are with indentation made in #6988 merge

**Comment by davorpa** [comment]
Some Vue & Svelte was lost. Maybe due to some bad merge. I'll review it later as last step previous to merge this PR

File: courses/free-courses-bn.md [hunk 9]
```diff
@@ -138,6 +145,16 @@
...
 
 
+#### Svelte
+
+* [Svelte 3: The Game Changer (Bangla Crash Course)](https://www.youtube.com/playlist?list=PL_XxuZqN0xVDJOOYnZxK-fbKxjxFHfc-H) - Stack Learner
+
+
+#### Vue
+
+* [Introduction to Vue.js in Bangla বাংলা  with a full Project - Vue Crash Course](https://www.youtube.com/playlist?&list=PLHiZ4m8vCp9O5_QoMvQbOAOR5WolyZ4OS) - Learn with Sumit
```
Comment (davorpa) [inline courses/free-courses-bn.md:155]: Revert to video url, which is 1h length enough

```suggestion
* [Introduction to Vue.js in Bangla বাংলা  with a full Project - Vue Crash Course](https://www.youtube.com/watch?v=iIvN7upsLoA) - Learn with Sumit
```

**Comment by davorpa** [comment]
> Some initial Vue & Svelte was lost. Maybe due to some bad merge in commits. I'll review it later as last step previous to merge this PR

This resources was removed due to haven't the minimum length of 1h

**Review by davorpa — APPROVED** [review]
Review chop is enough for me

**Review by LuigiImVector — APPROVED** [review]
LGTM

**Pull request merged**